express-notes-api/server.js

```diff
@@ -1,104 +1,105 @@
 import express from 'express';
 import * as fs from 'node:fs/promises';
 
-/* This just makes it easier to create error messages for me.
-   And it looks cleaner when written in the code later on. */
-class Error {
-  constructor(errorMsg) {
-    this.error = errorMsg;
-  }
-}
+const error = (message) => ({ error: message });
 
 const port = 8080;
+/**
+  I am used to having to read and write to different files from my Computational Linguistics work.
+  While this api will only ever read and write to one file, I always found it helpful to not hard code
+  where I am reading and writing in case I ever re-used code in another program or I needed to try a
+  different approach and change where I was recording the outcome of my program. Additionally, I would
+  have many different files to read from, and having one variable at the top that held where I was reading
+  from made it easier since I was often retrieving where I wanted to read from on the command line or in another file,
+  and so I couldn't hard code it and had to pass the info to my read and write functions. So it's just 'muscle memory' from Comp Ling.
```

Comment:
That's fine. More explanation than necessary! :-)

This will all be moot soon, anyway, when we switch to using the database instead of a local file.